app/views/consultations/show.html.erb

```diff
@@ -35,8 +35,7 @@
 
           <%= render partial: "document_summary", locals: { document: @document } %>
         </div>
-      <% else %>
-
+      <% elsif @document.closed? %>
```

Comment:
The catch all case contains language that is specific to an unopened consultation. I think we should have a specific case for `@document.unopened?`